.github/workflows/message-release.yml

```
@@ -29,6 +29,7 @@ jobs:
               "- [ ] Version correctly updated in the Pyproject",
               "- [ ] Changelog updated with the new version",
               "- [ ] Target a branch with name 'main' or 'release/**'",
+              "- [ ] Update accordingly the EOL page",
```

I think that this should be done on PRs, same logic as the changelogs in the end so maybe the phrasing could be:
```suggestion
              "- [ ] End-of-life table updated",
```